source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -380,14 +395,19 @@ export default class WalletRestoreDialog extends Component<Props, State> {
 
     const regularTabClasses = classnames([
       'regularTab',
-      !this.isCertificate() ? styles.activeButton : '',
+      !this.isYoroi() && !this.isCertificate() ? styles.activeButton : '',
     ]);
 
     const certificateTabClasses = classnames([
       'certificateTab',
       this.isCertificate() ? styles.activeButton : '',
     ]);
 
+    const yoroiTabClasses = classnames([
+      'yoroiTab',
+      this.isYoroi() && !this.isCertificate() ? styles.activeButton : '',
```

Comment:
@DeeJayElly here it should be enough to use `this.isYoroi()` - I don't think we need the `&& !this.isCertificate()` part...

Reply:
Fixed